client/app/scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -270,6 +270,59 @@ function MetadataListEditingDirective() {
     };
 }
 
+MetadataSliderDirective.$inject = ['desks'];
+function MetadataSliderDirective(desks) {
+    return {
+        scope: {
+            list: '=',
+            disabled: '=ngDisabled',
+            item: '=',
+            field: '@',
+            change: '&'
+        },
+        templateUrl: 'scripts/superdesk-authoring/metadata/views/metadata-slider.html',
+        link: function(scope) {
+            desks.initialize().then(function() {
+                var maxValue = scope.list.length,
+                    currentValue = scope.item[scope.field];
+
+                $('.sd-slider').slider({
+                    range: 'max',
+                    min: 0,
+                    max: maxValue,
+                    value: currentValue,
+                    create: function () {
+                        $(this).find('.ui-slider-thumb').css('left', (currentValue * 100) / maxValue + '%');
+                    },
+                    slide: function (event, ui) {
+                        $(this).find('.ui-slider-thumb').css('left', (ui.value * 100) / maxValue + '%').text(ui.value);
+                        scope.select(scope.list[ui.value - 1]);
+                    },
+                    start: function () {
+                        $(this).find('.ui-slider-thumb').addClass('ui-slider-thumb-active');
+                    },
+                    stop: function () {
+                        $(this).find('.ui-slider-thumb').removeClass('ui-slider-thumb-active');
+                    }
+                });
+            });
+
+            scope.select = function (item) {
```

Comment:
:+1:  First I plan to use as attached on tag, but this slider does not responds on ng-change :D ... I forgot to remove it from scope.. :)